### PR TITLE
Update GRASS integration.rst

### PR DESCRIPTION
**Pull request by dragons8mycat**
Minor fixes to GRASS integration typos and grammar

### CODE REVIEW

**Comment by dragons8mycat** [comment]
Tried creating a branch....still tries to pull in the older "update to qgis" pull request which was closed and deleted a while ago. 
Not sure what to do from here, as with the right access I could delete the file causing issue. I'm struggling to find a way of updating just the one .rst so that it ignores the others.....surely there must be  away!

**Comment by yjacolin** [comment]
Wondering if your local repository is out of date. Did you merge  upstream:master into your local master regurlarly? If no I will try to write you how to do it this  week end.

**Comment by yjacolin** [comment]
@dragons8mycat could you check this 'how-to contribute to QGIS-Documentation" and check that you've done all the step described (especially the one on add a new remote repository): https://github.com/yjacolin/QGIS-Documentation/wiki/howto_contribute_QGIS-Doc

Thanks.

**Comment by dragons8mycat** [comment]
@yjacolin  Checked my local and it is all up to date, as you can see I created and created a pull request from the branch also....the message I am now getting is that I don't have access to merge the pull request. 
"This branch has conflicts that must be resolved
Only those with write access to this repository can merge pull requests."

File: source/docs/user_manual/working_with_vector/expression.rst [hunk 1]
```diff
@@ -10,7 +10,7 @@ Expressions
 .. contents::
    :local:
 
-The **Expressions** feature is available from many parts in |qg|. It can be 
+The **Expressions** feature is available from many parts in [qg]. It can be
```
Comment (yjacolin) [inline source/docs/user_manual/working_with_vector/expression.rst]: |qg| is variable and should not be changed in [qg].

File: source/docs/user_manual/working_with_vector/expression.rst [hunk 2]
```diff
@@ -147,7 +147,7 @@ This group contains functions to handle conditional checks in expressions.
 
 * Send back a value if the first condition is true, else another value::
 
-    CASE WHEN "software" LIKE '%QGIS%' THEN 'QGIS' ELSE 'Other'
+    CASE WHEN "software" LIKE '%[qg]%' THEN '[qg]' ELSE 'Other'
```
Comment (yjacolin) [inline source/docs/user_manual/working_with_vector/expression.rst]: "QGIS" here is an example of string, it should be kept as it is.

File: source/docs/user_manual/working_with_vector/expression.rst [hunk 3]
```diff
@@ -619,8 +619,7 @@ Your function will be implemented in the :guilabel:`Custom` functions group of
 the :guilabel:`Expression` tab after using the :guilabel:`Run Script` button.
 
 Further information about creating Python code can be found on 
-http://www.qgis.org/html/en/docs/pyqgis_developer_cookbook/index.html.
+http://www.qgis.org/html/en/docs/py[qg]_developer_cookbook/index.html.
```
Comment (yjacolin) [inline source/docs/user_manual/working_with_vector/expression.rst]: same as before, don't change qgis to [qg]

**Comment by dragons8mycat** [comment]
Okay, I can see what is happening here, the first commit related to Vector expressions (which was fixed 2 weeks ago) is trying to commit itself again even though I closed it, this causing the error with the GRASS integration doc. 
Will try to delete the vector commit.....again....*groan*....its like a ghost in the machine!

**Pull request closed without merging**